### DISCUSSION TITLE
Lidl Parkside Zigbee Smart Home PSBZS A1

### DISCUSSION THREAD

**Discussion by SzymonZy** [Koenkk/zigbee2mqtt#11994] [Devices]
Hello
Does zigbee2mqtt fully support this device? Is it able to unlock valve after temperature fall below 5 C? ZHA can't so you have to remove batteries and plug again to solve it. Has anyone used this device?

**Comment by hellvideorazor** [comment]
Hi,
I guess I do have exactly this issue. Battery removal is needed alter sub 5 degree temperature. How to proceede for debug?
